be/src/olap/storage_engine.h

```diff
@@ -296,7 +296,9 @@ class StorageEngine {
     static StorageEngine* _s_instance;
 
     Mutex _gc_mutex;
-    std::unordered_map<std::string, RowsetSharedPtr> _unused_rowsets;
+    // map<rowset_id(str), pair<rowset_path(str),RowsetSharedPtr>>
+    // if we use RowsetId as the key, we need custom hash func
+    std::unordered_multimap<std::string, std::pair<std::string,RowsetSharedPtr>> _unused_rowsets;
```

Comment:
Previous, rowset_id is not unique. 
Nowadays, rowset_id is unique. So you can use std::unordered_map<std::string, RowsetSharedPtr> _unused_rowsets instead, not use unique_id().

Reply:
It couldn't be better. I'll fix it.